### ISSUE TITLE
Request for details on traininig I3D

### ISSUE THREAD

**Issue by AlexHu123** [wanjinchang/I3D_Master#2]
Dear Jinchang,
Have you tried training from scratch on Kinetics? How do you initialize weights and batch norm params?
Thanks in advance!

**Comment by wanjinchang** [comment]
Oh, I donnot train from scratch on Kinetics.And I train the model from scratch on my own dataset! If you want to finetune you model,I think you can parse the kinetics-i3d pretrained model to initialize weights params.
